dartsim/src/JointFeatures.hh

```diff
@@ -52,7 +52,10 @@ struct JointFeatureList : FeatureList<
   GetPrismaticJointProperties,
   AttachPrismaticJointFeature,
 
-  SetJointVelocityCommandFeature
+  SetJointVelocityCommandFeature,
+  SetJointPositionLimitsCommandFeature,
```

Comment:
The `Command` suffix in `SetJointVelocityCommand` was used to differentiate it from `SetJointVelocity`. Since there are no existing joint limit functions, I think we can drop the `Command` suffix for the new functions and features you've added here.

Reply:
I did the requested renaming.